rds_database_cluster/rds_database_cluster.tf

```diff
@@ -10,24 +14,52 @@ resource "aws_db_subnet_group" "db_subnet_group" {
   tags        = "${var.subnet_group_tags}"
 }
 
-resource "aws_ssm_parameter" "db_root_master_username" {
+resource "aws_kms_key" "rds_cluster_encryption_key" {
```

Comment:
Not sure about creating the KMS key in the module. I put it there to know your inputs.